### DISCUSSION TITLE
Is there an option to print preview of any text or pdf content ?

### DISCUSSION THREAD

**Discussion by srinath-sankaragallu** [zauberzeug/nicegui#1400] [Ideas / Feature Requests]
If I want to prepare some text template or any bills, application form fill ups using nice gui I would like have a print preview of my docs

**Comment by falkoschindler** [comment]
Sorry, I don't understand the use case. Can't you use the print preview of your browser? How is this related to NiceGUI?

**Comment by rodja** [comment]
You could also take the content from the forms, build a pdf with some other python lib or generate an image and show this to the user again by adding an `ui.image` pointing to your preview.

**Comment by rodja** [comment]
I've created an example which takes some input fields to build a pdf and shows a live preview on the website: https://github.com/zauberzeug/nicegui/pull/1402
